.storybook/utils/argTypes/specialCharacters.js

```diff
@@ -0,0 +1,48 @@
+export const singleSpecialCharacters = {
+  options: [
+    "minus",
+    "questionMark",
+    "hash",
+    "at",
+    "dollar",
+    "percent",
+    "caret",
+    "exclamation",
+    "asterisk",
+  ],
+  mapping: {
+    minus: "-",
+    questionMark: "?",
+    hash: "#",
+    at: "@",
+    dollar: "$",
+    percent: "%",
+    caret: "^",
+    exclamation: "!",
+    asterisk: "*",
+  },
+};
+
+export const email = {
+  options: ["email"],
+  mapping: {
+    email: "Andrew.Tait@sage.com",
```

Comment:
Let's use a pretend email instead of Andrews - he left Sage a couple of years ago!

Reply:
I agree with Sam, @DlgSHi can we prepare generic email for cypress?

Reply:
Yes we can, but we need to use some real email address to make possible to confirm, but what is the issue with using an old employee email ?

Reply:
Sorry I'm confused, make it possible to confirm what? 

We shouldn't really use any employee details in test data, current or old!

Reply:
sorry my bad, we need to have an existing email to make possible to create email gravatar. To make it possible we need to have an existing email to confirm it after registration in https://en.gravatar.com

Reply:
So can we create a test one that can be used to do that instead @DlgSHi?